<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabSprite</class>
 <widget class="QWidget" name="TabSprite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="sprite_view"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sprite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sprite_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>Img</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_sprite_button">
       <property name="toolTip">
        <string>Save Sprite</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/device-floppy.svg</normaloff>:/icons/device-floppy.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_sprite_button">
       <property name="toolTip">
        <string>Copy Sprite</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/clipboard.svg</normaloff>:/icons/clipboard.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="plane_scroll">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>224</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>228</height>
       </rect>
      </property>
      <widget class="QLabel" name="plane_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sprite Plane</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Plane</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hide_button">
       <property name="text">
        <string>Hide Selected</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/eye-off.svg</normaloff>:/icons/eye-off.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="show_button">
       <property name="text">
        <string>Show Selected</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/eye.svg</normaloff>:/icons/eye.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_plane_button">
       <property name="toolTip">
        <string>Save Sprite</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/device-floppy.svg</normaloff>:/icons/device-floppy.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copy_plane_button">
       <property name="toolTip">
        <string>Copy Sprite</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
       <property name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/clipboard.svg</normaloff>:/icons/clipboard.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
